src/KafkaFlow.Serializer.Json/KafkaFlow.Serializer.Json.csproj

```diff
@@ -9,10 +9,13 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Confluent.SchemaRegistry" Version="1.6.3" />
+    <PackageReference Include="Confluent.SchemaRegistry.Serdes.Json" Version="1.6.3" />
     <PackageReference Include="System.Text.Json" Version="4.7.2" />
   </ItemGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\KafkaFlow.Abstractions\KafkaFlow.Abstractions.csproj" />
+    <ProjectReference Include="..\KafkaFlow.Serializer\KafkaFlow.Serializer.csproj" />
```

Comment:
Please see [this](https://github.com/Farfetch/kafka-flow/pull/140/files#r615706263).

Reply:
Same reason as before: one possible solution to the problem is to have a single project for all confluent based serializer

Reply:
Yes, I think we can have a dedicated Kafkaflow.Serialization.Confluent project to be responsible for registering the confluent based serializers.

Basically, it would have only the 'WithSchemaRegistry' method, at least for now.